opennlp-tools/src/main/java/opennlp/tools/sentdetect/SentenceDetectorME.java

```diff
@@ -55,6 +58,19 @@ public class SentenceDetectorME implements SentenceDetector {
    * Constant indicates no sentence split.
    */
   public static final String NO_SPLIT = "n";
+  
+  /**
+   * Constant indicates no sentence split.
+   */
+  private static final Pattern dateGermanFormatPattern = Pattern.compile("(?<!d)(?:(?:(?:0?[1-9]|"
```

Comment:
This is quite some regular expression! Might be worth a bit more explanation in the comment

Reply:
That's very true 😅. I'll add a nice explanation.